tests/test_apig_wsgi.py

```diff
@@ -349,6 +368,16 @@ def test_plain_header_single(simple_app):
     assert simple_app.environ["HTTP_TEST_HEADER"] == "foobar"
 
 
+def test_plain_header_single_multiple(simple_app):
```

Comment:
I've removed this test - it wasn't necessary. The `make_event` function inside the test suite collapses multiple values to one so it wasn't really testing anything different to `test_plain_header_single`.